cpu/cortexm_common/include/core_cm0.h

```diff
@@ -39,8 +39,8 @@
  #pragma system_include  /* treat file as system include file for MISRA check */
 #endif
 
-#ifndef __CORE_CM0_H_GENERIC
-#define __CORE_CM0_H_GENERIC
+#ifndef CORE_CM0_H_GENERIC
+#define CORE_CM0_H_GENERIC
```

Comment:
You mean how to see what went wrong with the travis build right?

You can click on [Details](https://travis-ci.org/RIOT-OS/RIOT/builds/109902000) in the CI "info-box" below to get an overview of your (this) build.
There you see a list of build jobs for this PR.
If you click one which is in progress or is already finished/failed you will be presented with the output of the specific build.
Usually the important information can be found at the bottom, giving a hint what actually went wrong.